assets/javascript/apps/pipeline/Page.tsx

```diff
@@ -54,7 +54,7 @@ export default function Page() {
             ) : (
               <>
                 <div className="text-lg font-bold">{name}</div>
-                {!readOnly && 
+                {!readOnly && origin !== "chatbots" &&
```

Comment:
We don't need to display the name at all if `origin == 'chatbots'` since it's already displayed in the breadcrumb.